lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -8,11 +8,11 @@ edition = "2018"
 crate-type = ["staticlib"]
 
 [dependencies]
-bitflags = "1.3.2"
+bitflags = "2.0.1"
```

Comment:
This major jump could cause breakages: https://github.com/bitflags/bitflags/releases/tag/2.0.0.

I think it's better to delegate it to someone more in-the-known and make it in a separate PR.

(Also 2.0.2 is already out.)